keyboards/bastardkb/scylla/readme.md

```diff
@@ -0,0 +1,22 @@
+# Scylla
+
+A modern, low-profile split ergonomic keyboard
+
+* Keyboard Maintainer: [Bastard Keyboards](https://github.com/Bastardkb/)
+* Hardware Supported: elite-C V4
+* Hardware Availability: [Bastard Keyboards](https://bastardkb.com/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make bastardkb/scylla:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+See the [keyboard build instructions](https://docs.bastardkb.com)
+
+
+## Important information regarding the reset
```

Comment:
Bootmagic lite is enabled, too.  So that's another way to get into the bootloader.  

But you may want to add the defines to add the "right" side keys too.

Reply:
Hey,

At the moment I don't have the time to look into implementing BM properly, I would also have to create/update my documentation, website, and advertisement.
The goal of this PR is to get a minimum viable software working for the owners of Scyllas.
I will look into it later if that's allright.

Thanks for the heads-up !